src/ssp.hpp

```diff
@@ -30,10 +30,12 @@ constexpr real betaPowerLaw = FL(1.0);
 #define SSP_CALL_ARGS x, t, ccpx, gradc, crr, crz, czz, rho, freq, ssp, iSegz, iSegr
 #define SSP_INIT_ARGS vec2 x, const real &fT, \
     LDIFile &ENVFile, PrintFileEmu &PRTFile, std::string FileRoot, \
-    SSPStructure *ssp, const AttenInfo *atten, const FreqInfo *freqinfo, HSInfo &RecycledHS, \
-    bool resetParams
-#define SSP_CALL_INIT_ARGS x, fT, ENVFile, PRTFile, FileRoot, ssp, atten, freqinfo, RecycledHS, \
-    resetParams
+    SSPStructure *ssp, const AttenInfo *atten, const FreqInfo *freqinfo, HSInfo &RecycledHS
+#define SSP_CALL_INIT_ARGS x, fT, ENVFile, PRTFile, FileRoot, ssp, atten, freqinfo, RecycledHS
+#define SSP_PARAM_INIT_ARGS vec2 x, const real &fT, \
+    PrintFileEmu &PRTFile, std::string FileRoot, \
+    SSPStructure *ssp, const AttenInfo *atten
+#define SSP_CALL_PARAM_INIT_ARGS x, fT, PRTFile, FileRoot, ssp, atten
```

Comment:
The reason I was doing this was because these sets of arguments were used repeatedly. If any of them is only used once (or they're different on each use), feel free to move them back to being normal parameters rather than defines.